websocketpp/transport/asio/endpoint.hpp

```diff
@@ -995,10 +993,10 @@ class endpoint : public config::socket_type {
 
     void handle_resolve(transport_con_ptr tcon, timer_ptr dns_timer,
         connect_handler callback, lib::asio::error_code const & ec,
-        lib::asio::ip::tcp::resolver::iterator iterator)
+        lib::asio::ip::tcp::resolver::results_type endpoints)
```

Comment:
Suggest renaming the parameter as `results` to match `results_type` instead of `endpoints`